eng/Packages.Data.props

```diff
@@ -108,9 +108,11 @@
   <ItemGroup Condition="'$(IsClientLibrary)' == 'true' and $(MSBuildProjectName.StartsWith('Microsoft.'))">
     <PackageReference Update="CloudNative.CloudEvents" Version="2.0.0" />
     <PackageReference Update="CloudNative.CloudEvents.SystemTextJson" Version="2.0.0" />
-    <PackageReference Update="Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson" Version="3.0.0" Condition="'$(TargetFramework)' == 'netcoreapp3.1'"/>
-    <PackageReference Update="Microsoft.Azure.Functions.Extensions" Version="1.0.0" />
-    <PackageReference Update="Microsoft.Azure.SignalR.Management" Version="1.11.0" />
+    <PackageReference Update="MessagePack" Version="1.9.11" />
```

Comment:
What is the package? We need to approve this before we can depend on it in one of our shipping libraries.

Reply:
According to subsequent modifications, MessagePack is not needed.
`Microsoft.AspNetCore.SignalR.Protocols.MessagePack` is used for serialization. It is used for tests now and I could move it to dependencies for test projects.
`Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson` is also for serialization. It is needed by SignalR extensions.
`Microsoft.Azure.SignalR.Management` is a core dependency used by SiganlR extension, hosted in https://github.com/Azure/azure-signalr/.
`Microsoft.Azure.SignalR.Serverless.Protocols` is hosted in the [old SiganlR extension repo](https://github.com/Azure/azure-functions-signalrservice-extension/), and might need to move to this repo later.

Reply:
If possible lets remove the dependency even from tests. We want to avoid any none Microsoft/Azure based packages dependences in the repo as much as possible.

Reply:
Currently they are all Microsoft/Azure based packages. But if we also move `Microsoft.Azure.SignalR.Serverless.Protocols` this project here, then seems `MessagePack` is unavoidable.

Reply:
Regardless of where Microsoft.Azure.SignalR.Serverless.Protocols lives we need to either get the that package dependency approved by the ArchBoard or we need to remove it.